rktlet/runtime/stage1.go

```diff
@@ -0,0 +1,31 @@
+package runtime
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/golang/glog"
+)
+
+func (r *RktRuntime) initializeStage1Image(ctx context.Context) error {
```

Comment:
I used the same naming as `runtime.initializeLoggingAppImage`, but I agree, I'll rename both.